### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by lgarciaaco**
Fixes https://github.com/syndesisio/syndesis/issues/4871 by:

 - Use existing sdk and operator images as default making the installation process easier. Move the part of the documentation where images are build to DEVELOPMENT seccion
 - Add missing properties in values.yaml: _resource_memory_limit_
 - Remove RBAC for default service account and add instructions on how to add admin role to mentioned account

### CODE REVIEW

File: operator/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM REPLACE_BASE_IMAGE
+FROM lgarciaac/helm-app-operator:v0.0.1
```
Comment (EricWittmann) [inline operator/Dockerfile]: Is this change OK?  `lgarciaac` seems too specific. :)

Comment (lgarciaaco) [reply operator/Dockerfile]: Sure, the right domain for this is _apicurito_ :), but i cant push there :(

Comment (lgarciaaco) [reply operator/Dockerfile]: @EricWittmann any chances to have those two images under apicurito? 

from `lgarciaac/helm-app-operator:v0.0.1` to `apicurito/helm-app-operator:v0.0.1`
and `lgarciaac/apicurito-operator` to `apicurito/apicurito-operator`

Comment (EricWittmann) [reply operator/Dockerfile]: I don't have an `apicurito` domain in docker hub but I do have `apicurio` - so how about we rename the images to:

* apicurio/apicurito-helm-app-operator
* apicurio/apicurito-operator

And I can happily push them to docker hub.

Comment (lgarciaaco) [reply operator/Dockerfile]: oh right I mean `apicurio` not `apicurito`. Please do so, retag and push the images to:
```
apicurio/apicurito-helm-app-operator
apicurio/apicurito-operator
```
And let me know so I test all over again and make appropriate changes in the code.

What about granting me the powers to do all this myself?? ;)

Comment (EricWittmann) [reply operator/Dockerfile]: Yeah that's probably better - what is your docker hub id?  Same as your GH id?

Comment (lgarciaaco) [reply operator/Dockerfile]: nope, it is that, `lgarciaac`

Comment (EricWittmann) [reply operator/Dockerfile]: Oh wow, I'm dumb.  :)  Of course it's `lgarciaac`!  lol

Comment (EricWittmann) [reply operator/Dockerfile]: OK I've granted you access to the Apicurio docker hub - go nuts!

Comment (EricWittmann) [reply operator/Dockerfile]: @lgarciaaco ^^^^^

Comment (lgarciaaco) [reply operator/Dockerfile]: Awesome!! Thx!

**Comment by lgarciaaco** [comment]
Oki, I tested the new images, I think we are good to merge

**Comment by EricWittmann** [comment]
Thanks!!

**Pull request merged**